src/platform/forms-system/src/js/state/helpers.js

```diff
@@ -1,4 +1,7 @@
-import _ from 'lodash/fp'; // eslint-disable-line no-restricted-imports
+import { dropRight, merge } from 'lodash';
+import dataGet from '../../../../utilities/data/get';
```

Comment:
I'm not sure why, but there's another `get` function in this file, so I had to think of a _clever_ alternative for this import.